fdbclient/include/fdbclient/GenericManagementAPI.actor.h

```diff
@@ -70,7 +70,8 @@ enum class ConfigurationResult {
 	SUCCESS_WARN_SHARDED_ROCKSDB_EXPERIMENTAL,
 	DATABASE_CREATED_WARN_ROCKSDB_EXPERIMENTAL,
 	DATABASE_CREATED_WARN_SHARDED_ROCKSDB_EXPERIMENTAL,
-	DATABASE_IS_REGISTERED
+	DATABASE_IS_REGISTERED,
+	ENCRYPTION_DATA_AT_REST_MODE_ALREADY_SET
```

Comment:
This ConfigurationResult is not being returned anywhere

Reply:
Seems like the fallout from my patch, intention was to catch any configuration changes in `changeConfig` and discourage the operation if done.